src/main/java/com/vinylteam/vinyl/service/DefaultProfileManagementService.java

```diff
@@ -43,7 +45,7 @@ public ChangePasswordResponse changeProfilePassword(HttpServletRequest request,
         userService.changeUserPassword(userProfileInfo, user);
         String token = jwtService
                 .authenticateByRequest(new LoginRequest(user.getEmail(), userProfileInfo.getNewPassword()))
-                .getToken();
+                .getJwtToken();
         logoutService.logout(request, null, null);
```

Comment:
seems to me that its not a good "interface" idea
maybe you should create logout method with 1 param, which under hood will send request logoutService.logout(request, null, null);